generators/cleanup.js

```diff
@@ -88,6 +88,10 @@ function cleanupOldFiles(generator) {
     if (generator.isJhipsterVersionLessThan('6.3.0') && generator.configOptions && generator.configOptions.clientFramework === 'react') {
         generator.removeFile('tslint.json');
     }
+
+    if (generator.isJhipsterVersionLessThan('6.4.0')) {
```

Comment:
Latest cleanups are with additional condition `&& generator.configOptions && generator.configOptions.clientFramework === 'angularX'` to avoid deleting files if `clientFramework` is not `angularX` and file with the same name exists. Can you add this condition also here?

Reply:
Done, thanks.